lib/api/tenure/v1/service.ts

```diff
@@ -77,7 +77,7 @@ export interface EditTenureParams extends Partial<TenureParams> {
   id: string;
   etag: string;
   tenuredAsset?: TenureAsset | null;
-  tempAccInfo?: TemporaryAccommodationInfo;
+  tempAccInfo?: Partial<TemporaryAccommodationInfo>;
```

Comment:
It has to be a whole object with all the currently available, otherwise this will result in data loss. For any nested models within the main model, treat this Patch as a Put.

Reply:
Thanks for the heads up